src/main/java/com/aws/greengrass/clientdevices/auth/iot/IotAuthClient.java

```diff
@@ -122,12 +145,58 @@ public boolean isThingAttachedToCertificate(Thing thing, Certificate certificate
             } catch (Exception e) {
                 logger.atError().cause(e).kv("thingName", thing.getThingName())
                         .kv("certificateId", certificate.getCertificateId())
-                        .log("Failed to verify certificate thing association. Check that the core device's IoT policy"
-                                + " grants the greengrass:VerifyClientDeviceIoTCertificateAssociation permission.");
+                        .log("Failed to verify certificate thing association. Check that the core device's "
+                                + "IoT policy grants the greengrass:VerifyClientDeviceIoTCertificateAssociation "
+                                + "permission.");
```

Comment:
I know this was there already, but remove the period at the end of the log. Our logger appends a period automatically.